bin/localstack/ses-verify-identity

```diff
@@ -1,36 +1,12 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 # DOC: Verify SES email identity in localstack.
 
-pushd $(git rev-parse --show-toplevel) > /dev/null
+source bin/lib.sh
+source bin/localstack/lib.sh
 
-set -e
+verify_identity_cmd="ses verify-email-identity \
+--region us-west-2 \
```

Comment:
I believe you don't need the trailing `\`s here

Reply:
done